src/analysis/retail/monk/mistweaver/normalizers/CastLinkNormalizer.ts

```diff
@@ -120,6 +121,21 @@ const EVENT_LINKS: EventLink[] = [
       return !HasRelatedEvent(linkingEvent, FROM_HARDCAST);
     },
   },
+  // From LC on target with Mists of Life talented
+  {
+    linkRelation: FROM_MISTS_OF_LIFE,
+    linkingEventId: [TALENTS_MONK.ENVELOPING_MIST_TALENT.id, SPELLS.RENEWING_MIST_HEAL.id],
+    linkingEventType: [EventType.ApplyBuff, EventType.RefreshBuff],
+    referencedEventId: TALENTS_MONK.LIFE_COCOON_TALENT.id,
+    referencedEventType: [EventType.Cast],
+    backwardBufferMs: 500,
+    additionalCondition(linkingEvent, referencedEvent) {
+      return (
+        !HasRelatedEvent(linkingEvent, FROM_HARDCAST) &&
+        !HasRelatedEvent(referencedEvent, FROM_HARDCAST)
+      );
+    },
```

Comment:
We should set the isActive property for these event links generated via talents
`isActive(c) {
  return c.hasTalent(TALENTS_MONK.MISTS_OF_LIFE);
}`